projects/iio_demo/src/main.c

```diff
@@ -192,7 +229,9 @@ int main(void)
 	iio_demo_out_init_par = (struct iio_demo_init_param) {
 		.name = demo_device_output,
 		.num_channels = 4,
+#ifdef XILINX_PLATFORM
 		.ddr_base_addr = DAC_DDR_BASEADDR,
```

Comment:
`ddr_base_addr` could be initialized with the address of the `static uint8_t g_buff[MAX_BUFF_SIZE];`, declared in main.c

Reply:
done